### PR TITLE
[Docs][Connector][Source][mysql]Added the mysql Connector Document version title Example pr

### PR DESCRIPTION
**Pull request by zhilinli123**
Issues:https://github.com/apache/seatunnel/pull/4803

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Review by lightzhao — APPROVED** [review]

**Comment by zhilinli123** [comment]
PTAL: @liugddx @ic4y

**Review by TaoZex — APPROVED** [review]

File: docs/en/connector-v2/sink/Mysql.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 > JDBC Mysql Sink Connector
 
+## Support Connector Version
```
Comment (ic4y) [inline docs/en/connector-v2/sink/Mysql.md]: The phrase 'Support Connector Version' is definitely not accurate. Consider using 'Support Mysql Version', 'Support Version', and so on.

**Review by liugddx — DISMISSED** [review]
+1

File: docs/en/connector-v2/sink/Mysql.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 > JDBC Mysql Sink Connector
 
+## Support Mysql Version
+
+- 5.5/ 5.6/5.7/8.0
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/Mysql.md]:
```suggestion
- 5.5/5.6/5.7/8.0
```

File: docs/en/connector-v2/source/Mysql.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 > JDBC Mysql Source Connector
 
+## Support Mysql Version
+
+- 5.5/ 5.6/5.7/8.0
```
Comment (Hisoka-X) [inline docs/en/connector-v2/source/Mysql.md]:
```suggestion
- 5.5/5.6/5.7/8.0
```

**Review by Hisoka-X — APPROVED** [review]

**Review by liugddx — APPROVED** [review]

**Pull request merged**